Common/src/CConfig.cpp

```diff
@@ -1058,6 +1058,8 @@ void CConfig::SetPointersNull(void) {
   Kind_TimeNumScheme = EULER_IMPLICIT;
 
   Gas_Composition = nullptr;
+  Supercatalytic_Wall_Composition = nullptr;
```

Comment:
Not needed

Reply:
It is used below.

Reply:
Sorry for the terseness, XxxxList options do this initialization themselves